doc/classifying-signatures.md

```diff
@@ -154,7 +154,7 @@ For more information on the value of this kind of comparison for
 metagenomics, please see the simka paper,
 [Multiple comparative metagenomics using multiset k-mer counting](https://peerj.com/articles/cs-94/),
 Benoit et al., 2016. Initial comparisons of metagenome similarity
-approximations computed with sourmash to the output of simka suggest a
+approximations calculated with sourmash to the output of simka suggest a
```

Comment:
Can we just say this without providing evidence? I have the evidence hiding somewhere on farm, but it does seem tangential to shove it into the docs. I am marginally nervous of it getting lost tho and some poor future grad student having to re-run simka.

Reply:
Sure, I removed the wording, but left the reference to simka ;)